### ISSUE TITLE
nav-link active class has no more css style

### ISSUE THREAD

**Issue by Lausselloic** [twbs/bootstrap#22470]
https://v4-alpha.getbootstrap.com/components/navs/#base-nav

Still in v4-dev branch, and doesn't fund any related issues about "active" class on "nav-link" in basic "nav" element. The first link have an active class which doesn't provide any style, 

Css is defined for .nav-link.active only for : 
.nav-tabs
.nav-pills
.navbar-light .navbar-nav
.navbar-inverse .navbar-nav

Looks like v4-dev never had a active class available for basic nav-link

Maybe need to update the doc, or the _nav.scss

**Comment by Lausselloic** [comment]
My fault, doesn't read the warning :-( 
I agree with the idea that basic nav won't have an active class. Maybe add the warning in the code in an html comment for guys like me who don't read plain text and read only html code : 
```
<a class="nav-link active" href="#">Active</a> <!-- No active states are included in the base nav, so active class can be remove -->
```

**Comment by Geczy** [comment]
+1

**Comment by patrickhlauke** [comment]
not a fan of adding those comments to all the examples. however, made the note about missing active states in base navs more prominent (and added further accessibility advice) here https://github.com/twbs/bootstrap/pull/22472

**Comment by Lausselloic** [comment]
Yes seen it.

Thanks

**Comment by a207052002** [comment]
Sorry, I have another issue about pill.(v4 alpha6)
And if it needs, I will put the demo out.
But here I tried to ask first.
First, the 'focus' or 'active' in nav-pills don't work well. It often loses its blue after clicking. And it turn correct color after I scroll the page.
And I tried to override the class('active', 'focus', 'hover' ) , and I found out that the pills only reacted correctly about hover...(Just like here said, I didn't read the documentation well)
I 'm such a noob that I can't dig out more.
But is there any other way to customize the color change of pill?
